graphs/adapter-script-generation/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/scriptgen/GraphScriptGenActionFactory.kt

```diff
@@ -22,6 +25,11 @@ object GraphScriptGenActionFactory : GraphActionFactory {
         override fun getActionScript() = GraphScriptingHelper.scriptifyFunction(GraphActions::create, graphName)
     }
 
+    private class CreateDirectedGraphScriptGenAction(graphName: String) :
```

Comment:
:no_entry_sign: you need to add a test to `GraphScriptGeneratorTest` when you do you'll realise you haven't handled the script gen for the declaration